deploy/crds/enterprise.splunk.com_indexerclusters_crd.yaml

```diff
@@ -681,14 +681,28 @@ spec:
               description: Full path or URL for one or more default.yml files, separated
                 by commas
               type: string
-            ephemeralStorage:
-              description: If true, ephemeral (emptyDir) storage will be used for
-                /opt/splunk/etc and /opt/splunk/var volumes
-              type: boolean
-            etcStorage:
-              description: Storage capacity to request for /opt/splunk/etc persistent
-                volume claims (default=”1Gi”)
+            defaultsUrlApps:
```

Comment:
I think we czan remove this from the indexercluster CRD as it is ignored there.

Reply:
As discussed this is auto-generated code and comes as a part of changes to common_types.go. Leaving this as is for now. In the future we can move defaultsUrlApps to CRD specific structures to avoid this.